### ISSUE TITLE
Toggle Switch - add focus ring to readonly state

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/boiler#792]
#### Description / User Story

Currently the Toggle Switch does not have a focus ring in the readonly state. This should be fixed.

----
#### Acceptance Criteria
- [x] When in readonly state, the Toggle Switch can be focused and when done so a focus ring will appear.

----
#### Background information
- ...
